Workbooks/WorkbooksMetadata.json

```diff
@@ -739,5 +739,44 @@
     "templateRelativePath": "SysmonThreatHunting.json",
     "subtitle": "",
     "provider": "Edoardo Gerosa"
+  },
+  {
+    "workbookKey": "VMwareCarbonBlackWorkbook",
+    "logoFileName": "vmwarecarbonblack_logo.svg",
+    "description": "Gain extensive insight into VMware Carbon Black Cloud - Endpoint Standard by analyzing, collecting and correlating Event logs.\nThis workbook provides visibility into Carbon Black managed endpoints and identified threat event",
+    "dataTypesDependencies": [ "CarbonBlackEvents_CL" ],
+    "dataConnectorsDependencies": [ "VMware Carbon Black" ],
```

Comment:
The data connector dependencies name cannot have spaces in the name- this needs to be equal to the data connector ID value you have in the data connector json file. Please fix this for all instances.

Reply:
corrections have been made, updated all data connector json files as well.